quiz.css

```diff
@@ -0,0 +1,48 @@
+.card {
+	width: 30%;
+	border: 1px solid lightblue;	
+	float: left;
+	margin: 1.5%;
+}
+
+.title {
+	border: 1px dotted lightgrey;
+	text-align: center;
+	padding: 10px 0;
+	margin-top: 0;	
+}
+
+img {
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+}
+
+.description {
+	text-align: justify;
+	margin: 5px 10px;
+}
+
+.availability {
+	background-color: darkgrey;
+	color: yellow;
+	padding-left: 10px;
+}
+
+.spec-price {
```

Comment:
Names are hard, but important.  think of a name that encompasses alllllll this class is used for.